.github/workflows/main.yml

```diff
@@ -22,6 +22,8 @@ jobs:
         run: |
           curl --location --fail https://github.com/bazelbuild/bazelisk/releases/download/v1.1.0/bazelisk-linux-amd64 --output /tmp/bazelisk
           chmod +x /tmp/bazelisk
+      - name: Unit Tests
```

Comment:
nit: Maybe "Starlark Unit Tests"? If we start adding more logic to wrapper/persistent worker it may become confusing

Reply:
Bazel calls these analysis tests, renamed and restructured to fit it.

Reply:
👍